runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/process.json

```diff
@@ -593,6 +599,7 @@
     "Process.Details.Modeler.ActionPanel.ActionsGroup.BeeOffice": "BeeOffice",
     "Process.Details.Modeler.ActionPanel.ActionsGroup.SAP": "SAP",
     "Process.Details.Modeler.ActionPanel.ActionsGroup.Application": "Aplikacja",
+    "Process.Details.Modeler.ActionPanel.ActionsGroup.Excel": "Excel Lokalnie",
```

Comment:
lets adjust it a bit

Reply:
is "Excel" fine? The sharepoint is called "SharePoint Excel"

other suggestions:
"Excel (aplikacja)"
"Excel (stacjonarnie)"

Reply:
I've noticed why actions prefixes are needed - while searching for "excel" the action "set cell value" is not visible because of missing prefix. For SharePoint Excel that works. 

Shouldn't we include action groups' names in searching results?